actors/abi/sector.go

```diff
@@ -1,9 +1,8 @@
 package abi
 
 import (
-	cid "github.com/ipfs/go-cid"
-
 	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	cid "github.com/ipfs/go-cid"
```

Comment:
Nit: Please follow the import ordering/grouping of `goimports -local` (or at least don't regress it).